agent-shell.el

```diff
@@ -531,6 +544,36 @@ LINE defaults to 1, LIMIT defaults to nil (read to end)."
                            :code -32603
                            :message (error-message-string err))))))))
 
+(defun agent-shell--resolve-path (path)
+  "Resolve PATH using `agent-shell-path-resolver-function'."
+  (funcall (or agent-shell-path-resolver-function #'identity) path))
+
+(defun agent-shell--get-devcontainer-workspace-path (cwd)
+  "Return devcontainer workspaceFolder for CWD; signal error if none found."
```

Comment:
Maybe a URL/link to what a devcontainer is would be handy for newcomers (including myself).

Reply:
Done 👍